modules/subchannel/doc/content/source/scmclosures/SCMHTCClosureBase.md

```diff
@@ -0,0 +1,118 @@
+- $h_{sc}(z)$ is the convective heat transfer coefficient for a subchannel $sc$ neighboring the pin at a height $z$.
+
+For the duct, the duct surface temperature is defined as follows:
+
+\begin{equation}
+T_{s,d}(z) = T_{bulk,d}(z) + \frac{q''_d(z)}{h_d(z)},
+\end{equation}
+
+where:
+
+- $T_{s,d}(z)$ is the duct surface temperature at a height $z$
+- $T_{bulk,d}(z)$ is the bulk temperature of the subchannel next to the duct node $d$
+- $q''_d(z)$ is the heat flux at the duct at a height $z$
+- $h_d(z)$ is the convective heat transfer coefficient for the subchannel next to the duct node at a height $z$
+
+In both cases, the heat exchange coefficients are computed using the Nusselt number (Nu) as follows:
+
+\begin{equation}
+h = \frac{\text{Nu} \times k}{D_h}
+\end{equation}
+
+where:
+
+- $k$ is the thermal conductivity of the subchannel neighboring the structure
+- $D_h$ is the hydraulic diameter of the subchannel neighboring the structure
+
+!alert note
+Currently there is no subchannel-to-duct heat transfer model implemented for square assemblies (square ducts). It only exists for hexagonal assemblies (hexagonal ducts). The subchannel-to-pin model is available for both hexagonal and square assemblies.
+
+### General Expression for the Nusselt number
+
+The laminar, turbulent and transition regimes use different coefficients for the Nusselt number.
+
+The bounding laminar and turbulent Reynolds numbers for the turbulent transition are defined as follows [!cite](chen2018upgraded):
+
+\begin{equation}
+Re_L = 320 \times 10^{(P/D_{\text{pin}} - 1.0)}
+\end{equation}
+
+\begin{equation}
+Re_T = 10^4 \times 10^{0.7 \times (P/D_{\text{pin}} - 1.0)}
+\end{equation}
+
+where:
+
+- P is the pitch
+- D$_{\text{pin}}$ is the pin diameter
+
+The flow is laminar if $Re \leq Re_L$, turbulent if $Re \geq Re_T$, and in the transition regime if it lies in between.
+The modeling of each regime is explained below.
+
+#### Laminar Nusselt Number
+
+The following relation is used depending on the subchannel:
+
+\begin{equation}
+\text{Nu}_{\text{laminar}} =
+\begin{cases}
+3.73 & \text{if subchannel is CENTER} \\
+3.59 & \text{if subchannel is EDGE} \\
+3.52 & \text{if subchannel is CORNER}
+\end{cases}
+\end{equation}
+
+The values defined here are chosen based on engineering judgement and the values for  laminar flow in a circular/square tube with constant surface temperature (3.66,3.091) and laminar flow in a circular/square tube subjected to constant surface heat flux (4.36,3.54). For a center subchannel that has approximately half contact with circular fuel pins and half flat interface the laminar Nusselt number is chosen to be the average of the two: $\text{Nu}_{\text{laminar}} = (4.36 + 3.091)/2 = 3.73$. Similar logic has been applied for the edge and corner subchannels. For the edge: $\text{Nu}_{\text{laminar}} = (3.54 + 2*4.36 + 3 * 3.091)/6 = 3.59$. For the corner: $\text{Nu}_{\text{laminar}} = (2*3.54 + 4.36 + 2*3.091)/5 = 3.52$
```

Comment:
This is iffy imo.

If you look at the T&K, figure 10.5 shows you the Nu for the rod array. And in that figure, they have the center channel at all q'' constant, not as a mix